prod-config.json.asc

```diff
@@ -1,19 +1,19 @@
 -----BEGIN PGP MESSAGE-----
 
-hQEMAy5VCdmwb1c1AQf8CmIfNVREkravi4bJZnx3AVEvZVBU5KMV8C2GYUeVs1zD
-JpNORgY+tpFXGK2gK/kDUXpHtr9vpWjqBMt5U+mQGq8024Ij0ZBfCu+rLmdY3CP8
-O39DUlmNS2C0W6bygXehN8MDXtSnZfryN+ZPVTxgn4YDaYhcqVy/MQBPPAfKnvFT
-jzzrHAouPjsZDlUGW7EpQRPb7JOdJN+NpHl875k267PTP7YSzwjfRfgrtpXAs4Y8
-/S/ZGfThXRvac5g+wpQD1I1cFXcZIss0Yg3FcwSR953IH0FDhe/P1NHBDGdLgxGj
-tI7ZYniLShnm0ZCg/1M8C2yoC4XIK8KTf4p2ko5notLA6QHfEVQ/UoYBsAPezKXs
-S/uGeZ8OmZf0tYxFs4P3XehyDiKx/XCFSVBX8mFkg0zYqJ6rusoOB03a0UB6M3fG
-xMcBgh+wC2KL+8mx5KEDcFYRDBI+UN9+PwtG4YYa0luWfEC5L5jgAZKGuSusf7JC
-nMEZOkI8jJlFDL5ow9koiAT9w9HhdEBI5ZvotScQ8ZR6WQlwgO6+m1E+RkZ3bC1g
-CNp1nuPCcQOHLPufnmj5XWp2TyWO9ToIGpzKgNlYZl4nhMqIxNAg9VnofLZC/bk0
-dxbu/CZO3Hc1tw6BUVwLx9k/f0ib+5Dun3HvgMhicoupMcg+LT+7Tk1cyaaAw8iI
-rGasi60G9rKKa3oogDXYa8IESp41U+crsz9OEW8Ko+EgEv8vVDuVxlLVChd48jiR
-GRhITtaxtkWJtubj9uSi0fadp6fsh4rqJqX8mM05fAWYp8qgLBHhs2wZGzoLQVWm
-Ant6fmdb6qVekXhKl9ubbQEgy4GzKefZpFrISq/HbgNcnOe888GhTq6d2vnTaJy8
-IYpUPIVmehGyYiTAF1SvLF1qBm+1pmqxZrEy
-=SGEl
+hQEMA4TG4yJX2eSjAQgAj5P3W/6iROk7/lb2yO3lPmy7fUhDsxavYBquu8OjGr8S
+RxP1ZFri4BZTY5dvtXew+iBem3esAojs22SKisq+AyXWWZlbBX09+rCvz9O08AMw
+JoLxDJVbx5vz5p9JFXD97HjoBjdQe4q4YtGATLHGLqx6fCAmm89gRn09mCRvCLVK
+YGjf1RK9jXu31e3LyPT049wNKBNMCDSzHpvQh4SaXD+wagTf4EV1sqZM7J2Q83Zi
+hTnqyUNeQ5HB6HPcZQbNNjdsRNm7R2kC1FBE2ABNWPn+87VOujuvp669DAynuZZ+
+k9pGvzT70trtvJMB3vTnRM5R9mFepqThiTAYz3J/udLA/AEraBgQ2CoS8H/9RdJz
+EerSgwbjSRd660Lq8oBqNI2lnxT6VVcupLVtcM21sWLnivjwVduIvy+ttOUbQPPa
+3EVI/HVKFknwC3TXCn90jqC4XE/eI4h3C3eiyrSb2uMcmHGgqvXRB/K8QoqQiN40
+lZr1l2izfRk8Cxiy4xm723k5EOtECRF9O/qOS8sZ5aO3WDmzJwmmFDV3i0CJ6ELp
+G0Gl3H0wSoO47HVIFtUApgUSM644mDgXO9Vk1PR7VgVk2kL7rxGvizJ82CnK2M9n
+RhV8N/Ks5HbrOSqzk3HA5WJIVYD/syXlrKz2Pg6PCfh/xoI/WQZiDZGvK067eL4S
+4hw3oS1wJKLHBF2UFbI4KBZ3EMW46H67fj9rDHvpzT6jazteZYB/HDncnG5DzNM+
+xEEJkISTN6KsdVZSMI9mSh+2wKDKn3lJCzn904pnTXf0QdgBpa81s/hZ1f//hWaa
+iREBMJQ+wVGud/4lF5J2e2K7Hlrr1XLNj0xwjMGk4PmJyBEBkS19Ui8LfDOdIV4y
+7moQ+UuMORZBniwTBBxAs0UtSy1OkH+dkOubgrY6DcUlN5AdCjnD6snfuE3Whg==
+=gG/+
```

Comment:
@rpapa: Actually, we may need to revert these as well... Looking at **master**, it seems like these were changed in your repo/PR. Not sure why [yet].